plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -270,18 +285,7 @@ public JdbcTableHandle getTableHandle(ConnectorSession session, PreparedQuery pr
         catch (SQLException e) {
             throw new TrinoException(JDBC_ERROR, "Failed to get table handle for prepared query. " + firstNonNull(e.getMessage(), e), e);
```

Comment:
Exception message is about table handle, but it's getColumns

Reply:
Fixed